cmd/lotus-shed/balances.go

```diff
@@ -514,7 +513,8 @@ var chainBalanceStateCmd = &cli.Command{
 		cst := cbor.NewCborStore(bs)
 		store := adt.WrapStore(ctx, cst)
 
-		sm, err := stmgr.NewStateManager(cs, consensus.NewTipSetExecutor(filcns.RewardFunc), vm.Syscalls(proofsffi.ProofVerifier), filcns.DefaultUpgradeSchedule(), nil, mds, index.DummyMsgIndex)
+		sm, err := stmgr.NewStateManager(cs, consensus.NewTipSetExecutor(filcns.RewardFunc), vm.Syscalls(proofsffi.ProofVerifier), filcns.DefaultUpgradeSchedule(), nil, mds,
+			nil)
```

Comment:
these `nil` on newlines bother me though, this linebreak seems a lot longer than the others so maybe you're just hitting enter manually?

Reply:
my personal preference when a line gets too long is to just break everything, both on the call and on functions definitions

```suggestion
		sm, err := stmgr.NewStateManager(
			cs,
			consensus.NewTipSetExecutor(filcns.RewardFunc),
			vm.Syscalls(proofsffi.ProofVerifier),
			filcns.DefaultUpgradeSchedule(),
			nil,
			mds,
			nil,
		)
```